draft-pauly-httpbis-geoip-hint.md

```diff
@@ -73,63 +107,91 @@ For example, the header for an entry "192.0.2.5,US,US-AL,Alabaster" would be:
     Sec-CH-IP-Geo = "US,US-AL,Alabaster"
 ~~~
 
-This field also defines a parameter, "feed", that contains the URI of the
-IP geolocation feed that is authoritative for this entry. For example:
-
-~~~
-    Sec-CH-IP-Geo = "SG,SG-01,Singapore"; feed="https://noc.ietf.org/geo/google.csv"
-~~~
-
-Servers that can provide different content based on Geohash hints SHOULD include
-the headers in their "Accept-CH" list.
+Given that the Sec-CH-IP-Geo is a high-entropy client hint, meaning it is
+a client hint that is not in the low-entropy hint table, the server
+MUST explicitly opt-in to receive the Geo Client Hint as defined in
+{{RFC8942}}. It will not be sent by default and the server MAY
+indicate support for this hint via the Accept-CH header in the
+initial response:
 
 ~~~
     Accept-CH: Sec-CH-IP-Geo
 ~~~
 
-Servers also SHOULD indicate for any cacheable content if the IP geo hints will influence
-the cached content, using the "Vary" header.
+Servers SHOULD indicate for any cacheable content if the geo hints
+will influence the cached content, using the 'Vary' header. This will
+indicate that the server may have used this header as a determining
+factor when choosing a response:
 
 ~~~
     Vary: Sec-CH-IP-Geo
 ~~~
 
-# Server Behavior
-
-Upon receiving a IP Geolocation Client Hint, a server can use the information to influence
-its behavior in various ways, such as determining the content of HTTP responses.
-
-Many servers have existing IP geolocation feeds that they use to identify client locations.
-Servers can choose to use the hint value in one of several ways:
+# Client Behavior
+ 
+The client MUST determine geolocation using a cooperating server
+that looks up the client's IP address in a geo-IP database. The client
+MUST NOT use GPS. The client hint value MUST NOT be more precise
+or detailed than what can be inferred from the user’s IP address.
+If the client cannot determine how much information its external
```

Comment:
I'm not sure what this requirement means, how do we quantify information?

Reply:
@tfpauly suggested this change in a previous comment - Tommy could you elaborate here?

Reply:
I think that Tommy's comment was before we decided to force the use of a cooperating server. I would just remove this sentence now